include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -997,6 +997,39 @@ def TTIR_ClampOp : TTIR_DPSOp<"clamp"> {
     let hasVerifier = 1;
 }
 
+def TTIR_ReverseOp : TTIR_DPSOp<"reverse", [AllShapesMatch<["input", "result"]>]> {
```

Comment:
Can we add a verifier for this op? Just to make sure that `dimensions` doesn't contain any out of bounds dims.

Reply:
Copied over verifier from stablehlo implementation. Check out `lib/Dialect/TTIR/IR/TTIROps.cpp`.

Seemes weird that they declared `dimensions` to be singed int, and yet forbid negative dimensions.